certbot-nginx/certbot_nginx/parser.py

```diff
@@ -468,6 +468,17 @@ def _is_include_directive(entry):
             len(entry) == 2 and entry[0] == 'include' and
             isinstance(entry[1], str))
 
+def _is_ssl_on_directive(entry):
+    """Checks if an nginx parsed entry is an 'include' directive.
```

Comment:
I think the first line of this docstring was wrongly left unchanged when pasting from elsewhere? Should be "is an 'ssl on' directive", matching the `returns` value right below.